app/controllers/api/v1/rates_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1
+    class RatesController < ApplicationController
+      before_action :set_recipe
+      def create
+        rate = Rate.new(rate_params)
+        rate.recipe = @recipe
+        if rate.save
```

Comment:
It's good pattern to use a "parent" object as a foundation to create an association.
In this case you can go with ```if @recipe.rates.create(rate_params)```